### ISSUE TITLE
Ability to have the coverage table only show coverage information for the files that changed in the PR.

### ISSUE THREAD

**Issue by sghsri** [MishaKav/jest-coverage-comment#14]
Hi, first of all I love this action! However, with a massive project with a lot of files, I think it would be awesome if there was a way to filter the coverage comment to only show coverage information for the files that were changed in the PR, rather than ALL files. Thanks!

**Comment by MishaKav** [comment]
I've already implemented this feature in `v1.0.6` (the current version is `v1.0.8`).
you can just pass `report-only-changed-files: true` [more-info](https://github.com/MishaKav/jest-coverage-comment#inputs)

example:
```yaml
- name: Jest coverage comment
  uses: MishaKav/jest-coverage-comment@main
  with:
    coverage-summary-path: ./coverage/coverage-summary.json
    coverage-path: ./coverage.txt
    report-only-changed-files: true
```

**Comment by sghsri** [comment]
oh ok thank you!!!
